improver/utilities/load.py

```diff
@@ -116,76 +106,57 @@ def load_cube(filepath, constraints=None, no_lazy_load=False,
     # describe probabilistic data)
     cubes = strip_var_names(cubes)
 
-    # Merge loaded cubes
-    if len(cubes) == 1:
-        cube = cubes[0]
-    else:
-        cube = MergeCubes()(cubes)
+    for cube in cubes:
+
+        # Remove metadata prefix cube attributes
+        if 'bald__isPrefixedBy' in cube.attributes.keys():
+            cube.attributes.pop('bald__isPrefixedBy')
+
+        # Ensure the probabilistic coordinates are the first coordinates within
+        # a cube and are in the specified order.
+        enforce_coordinate_ordering(
+            cube, ["realization", "percentile", "threshold"])
+        # Ensure the y and x dimensions are the last within the cube.
+        y_name = cube.coord(axis="y").name()
+        x_name = cube.coord(axis="x").name()
+        enforce_coordinate_ordering(
+            cube, [y_name, x_name], anchor_start=False)
+        if no_lazy_load:
+            # Force cube's data into memory by touching the .data attribute.
+            # pylint: disable=pointless-statement
+            cube.data
 
-    # Remove metadata prefix cube attributes
-    if 'bald__isPrefixedBy' in cube.attributes.keys():
-        cube.attributes.pop('bald__isPrefixedBy')
-
-    # Ensure the probabilistic coordinates are the first coordinates within a
-    # cube and are in the specified order.
-    enforce_coordinate_ordering(
-        cube, ["realization", "percentile", "threshold"])
-    # Ensure the y and x dimensions are the last dimensions within the cube.
-    y_name = cube.coord(axis="y").name()
-    x_name = cube.coord(axis="x").name()
-    enforce_coordinate_ordering(
-        cube, [y_name, x_name], anchor_start=False)
-    if no_lazy_load:
-        # Force the cube's data into memory by touching the .data attribute.
-        # pylint: disable=pointless-statement
-        cube.data
-    return cube
+    return cubes
 
 
-def load_cubelist(filepath, constraints=None, no_lazy_load=False):
-    """Load one cube from each of the filepath(s) provided using Iris into
-    a cubelist.
+def load_cube(filepath, constraints=None, no_lazy_load=False):
+    """Load the filepath provided using Iris into a cube.  Strips off all
+    var names except for "threshold"-type coordinates, where this is different
+    from the standard or long name.
```

Comment:
Need to move the var_names bit of the docstring into `load_cubelist`.

Reply:
OK, copied instead as this is relevant to both of them.